src/openrct2/network/NetworkBase.cpp

```diff
@@ -912,9 +911,9 @@ uint8_t NetworkBase::GetGroupIDByHash(const std::string& keyhash)
     const NetworkUser* networkUser = _userManager.GetUserByHash(keyhash);
 
     uint8_t groupId = GetDefaultGroup();
-    if (networkUser != nullptr && networkUser->GroupId.HasValue())
+    if (networkUser != nullptr && networkUser->GroupId)
```

Comment:
We generally use explicit comparisons for anything that is not a boolean. E.g.
```suggestion
    if (networkUser != nullptr && networkUser->GroupId.has_value())
```